SECURITY.md

```diff
@@ -1,72 +1,148 @@
+* Work with you to understand, resolve and ultimately disclose the issue in a timely fashion 
+
+## Disclosure Process 
+Tendermint Core uses the following disclosure process:
+
+1. Once a security report is received, the Tendermint Core team works to verify the issue and confirm its severity level using CVSS. 
+2. The Tendermint Core team collaborates with the Gaia team to determine the vulnerability’s potential impact on the Cosmos Hub. 
+3. Patches are prepared for eligible releases of Tendermint in private repositories. See “Supported Releases” below for more information on which releases are considered eligible. 
+4. If it is determined that a CVE-ID is required, we request a CVE through a CVE Numbering Authority. 
+5. We notify the community that a security release is coming, to give users time to prepare their systems for the update. Notifications can include forum posts, tweets, and emails to partners and validators, including emails sent to the [Tendermint Security Mailing List](https://berlin.us4.list-manage.com/subscribe?u=431b35421ff7edcc77df5df10&id=3fe93307bc).
+6. 24 hours following this notification, the fixes are applied publicly and new releases are issued. 
+7. Cosmos SDK and Gaia update their Tendermint Core dependencies to use these releases, and then themselves issue new releases. 
+8. Once releases are available for Tendermint Core, Cosmos SDK and Gaia, we notify the community, again, through the same channels as above. We also publish a Security Advisory on Github and publish the CVE, as long as neither the Security Advisory nor the CVE include any information on how to exploit these vulnerabilities beyond what information is already available in the patch itself. 
+9. Once the community is notified, we will pay out any relevant bug bounties to submitters. 
+10. One week after the releases go out, we will publish a post with further details on the vulnerability as well as our response to it. 
+
+This process can take some time. Every effort will be made to handle the bug in as timely a manner as possible, however it's important that we follow the process described above to ensure that disclosures are handled consistently and to keep Tendermint Core and its downstream dependent projects--including but not limited to Gaia and the Cosmos Hub--as secure as possible. 
+
+### Example Timeline 
+The following is an example timeline for the triage and response. The required roles and team members are described in parentheses after each task; however, multiple people can play each role and each person may play multiple roles. 
+
+#### > 24 Hours Before Release Time
+
+1. Request CVE number (ADMIN) 
+2. Gather emails and other contact info for validators (COMMS LEAD) 
+3. Test fixes on a testnet  (TENDERMINT ENG, COSMOS ENG) 
+4. Write “Security Advisory” for forum (TENDERMINT LEAD) 
+
+#### 24 Hours Before Release Time
+
+1. Post “Security Advisory” on forum (TENDERMINT LEAD) 
```

Comment:
I think you refer to posting a sort "secutiy patch is coming' announcement, and not the whole advisory content, don't you?

Reply:
Yes! Good clarification; I'll add that